src/core/worker.js

```diff
@@ -544,6 +544,7 @@ class WorkerMessageHandler {
           pdfManager.ensureCatalog("acroForm"),
           pdfManager.ensureCatalog("acroFormRef"),
           pdfManager.ensureDoc("startXRef"),
+          pdfManager.ensureDoc("isLinearized"),
```

Comment:
As mentioned above, I think that we can just do the following instead (with suitable changes below as well).
```suggestion
          pdfManager.ensureDoc("linearization"),
```